recipes/packages.rb

```diff
@@ -16,8 +16,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-node['osl-php']['packages'].each do |p|
-  package p
+if node['osl-php']['packages'].any?
+  node.normal['php']['packages'] = node['osl-php']['packages']
```

Comment:
@ramereth So this isn't related to upgrading the version, but what do you think about this approach? I think it makes more sense, considering `php::package` does the same thing with the `node['php']['packages']` attribute as what we're doing here with our wrapper attribute.

In other cookbooks (like in `proj-deluge::php` and `proj-misc::web2`) we were clearing the `node['php']['packages']` attribute to keep it from conflicting with the packages we specified in `node['osl-php']['packages']`. Forgetting to do so results in conflicting packages.

Instead of having to remember to clear it whenever we want to use this recipe, this either uses the packages we specify or defaults to the upstream packages in the `php` cookbook.

From what I've seen, we don't have a use case where we specify packages for the wrapper's attribute without clearing `php`'s attribute, so that shouldn't be a problem.

Reply:
I don't remember why we did it this way originally but this makes sense. We just need to test and make sure this doesn't break the other recipes that use it.